assests/styling/menu.scss

```diff
@@ -171,126 +171,9 @@ html, body{
     margin-left: -15rem;
 }
 
-.page__content {
-    width: 100%;
-    padding-right: 5rem;
-}
```

Comment:
.page__content is om te zorgen dat het submenu nooit in de tekst gaat. Als je deze weglaat kan het submenu icon door te tekst gaan.